jobs/ci-kubernetes-e2e-gce-scalability.env

```diff
@@ -13,6 +13,8 @@ NUM_NODES=100
 ALLOWED_NOTREADY_NODES=1
 REGISTER_MASTER=true
 
+# Switch off image puller to workaround #44701
```

Comment:
Done. I've changed it for all tests with (NUM_NODES >= 100 && ! previous release && ! gke && ! kubemark).
Because it doesn't affect our kubemark tests and for gke it appears like this prepuller is not mentioned anywhere in the config-(default/test),sh files.